mate-panel/panel-run-dialog.c

```diff
@@ -1945,6 +1993,26 @@ key_press_event (GtkWidget    *run_dialog,
 	return FALSE;
 }
 
+static void panel_run_dialog_accelerator_key_pressed(GtkAccelGroup *accel_group, GObject *acceleratable,
+                                               guint keyval, GdkModifierType modifier, PanelRunDialog *dialog)
```

Comment:
style: specifiers and return type on its own line, and each argument on one line with both type and name aligned

Reply:
Have a look again, I tried to fix the style.